### PR TITLE
Improve landing page

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

File: _pages/index.md [hunk 1]
```diff
@@ -3,53 +3,94 @@ layout: landing
...
-		<div class="container has-text-centered">
-			<div class="column is-6 is-offset-3 value-proposition">
-				<h1 class="title">
+<div>
+	<section class="hero is-medium is-bold">
+		<div class="hero-head">
+			<header class="navbar">
+				<div class="container">
+					<div class="navbar-brand">
+						<img class="logo" src="https://avatars3.githubusercontent.com/u/7874148?s=400&v=4" alt="Logo">
```
Comment (MattiSG) [inline _pages/index.md]: Either empty `alt` if the image is decorative or a meaningful alt.
```suggestion
						<img class="logo" src="https://avatars3.githubusercontent.com/u/7874148?s=400&v=4" alt="">
```

File: _pages/index.md [hunk 2]
```diff
@@ -3,53 +3,94 @@ layout: landing
...
-	<div class="hero-body">
-		<div class="container has-text-centered">
-			<div class="column is-6 is-offset-3 value-proposition">
-				<h1 class="title">
+<div>
+	<section class="hero is-medium is-bold">
+		<div class="hero-head">
+			<header class="navbar">
+				<div class="container">
+					<div class="navbar-brand">
```
Comment (MattiSG) [inline _pages/index.md]: > So many `<div>`…  😞

File: _pages/index.md [hunk 3]
```diff
@@ -3,53 +3,94 @@ layout: landing
...
+						<img class="logo" src="https://avatars3.githubusercontent.com/u/7874148?s=400&v=4" alt="Logo">
+					</div>
+					<div class="navbar-menu">
+						<div class="navbar-end">
+							<a href="/collaborate" class="navbar-item" data-proofer-ignore>
```
Comment (MattiSG) [inline _pages/index.md]: Add a comment to explain this `data-proofer-ignore` attribute.

File: _pages/index.md [hunk 4]
```diff
@@ -3,53 +3,94 @@ layout: landing
...
+								Collaboration
+							</a>
+							<a href="/encyclopedia" class="navbar-item">
```
Comment (MattiSG) [inline _pages/index.md]: Shouldn't this be prefixed with `{{ baseurl }}` in order to allow deployments that don't make the assumption of being hosted at root?

File: _pages/index.md [hunk 5]
```diff
@@ -3,53 +3,94 @@ layout: landing
...
+								Encyclopedia
+							</a>
+							<a href="/encyclopedia" class="navbar-item">
+								About
+							</a>
```
Comment (MattiSG) [inline _pages/index.md]: Let's remove this link until we have a proper `about` page, right now this is misleading.

File: _pages/index.md [hunk 6]
```diff
@@ -3,53 +3,94 @@ layout: landing
...
+				What we're currently working on…
+			</h1>
 			<div class="columns">
 				<div class="column is-5 is-offset-1">
-					<div class="box">
-						<h2 class="title">
-							Collaboration chat
+					<div class="work-item">
+						<h2 class="title is-spaced">
+							<span class="thin">Collaboration</span>chat
```
Comment (MattiSG) [inline _pages/index.md]: Add a space between words. #a11y #seo
```suggestion
							<span class="thin">Collaboration</span> chat
```

File: _pages/index.md [hunk 7]
```diff
@@ -3,53 +3,94 @@ layout: landing
...
 						</h2>
-						<h4 class="subtitle mt-1">
-							We aim at reinforcing the democratic antibodies that are present in our societies.
-						</h4>
 						<p>
-							We believe that in the fight against disinformation, many actors are already doing their part to prevent fake news and information manipulation from having an impact on our elections and collective decision processes: investigative journalists, fact-checkers, public open data providers, NGOs, OSINT researchers, concerned citizens…
+							Our main focus at the moment is a chat on steroids: a real-time communication channel where all detection, analysis and reaction can happen in a shared fashion with mutualized tools. You’ll find more about the chatbots we have built <a href="/encyclopedia/chatbots/">here</a>. If you’re reading this we’d probably like to have you on board. <a href="mailto:matti.schneider@diplomatie.gouv.fr">Send us an email!</a>
```
Comment (MattiSG) [inline _pages/index.md]:
```suggestion
							Our main focus at the moment is a chat on steroids: a real-time communication channel where all detection, analysis and reaction can happen in a shared fashion with mutualized tools. Learn more about the <a href="/encyclopedia/chatbots/">chatbots we have built</a>. If you’re reading this we’d probably like to have you on board. <a href="mailto:matti.schneider@diplomatie.gouv.fr">Send us an email!</a>
```

Comment (MattiSG) [inline _pages/index.md]: Shouldn't `baseurl` be somewhere?

File: _pages/index.md [hunk 8]
```diff
@@ -3,53 +3,94 @@ layout: landing
...
 		</div>
-	</div>
-</section>
+	</section>
+	<footer class="footer">
+		<div class="container has-text-centered">
+			<div class="columns">
+				<div class="column is-full">
+					<a href="/encyclopedia">About</a>
+					&nbsp;|&nbsp;
```
Comment (MattiSG) [inline _pages/index.md]: > This would be more appropriate as `border-right`.

File: _sass/custom/custom.scss [hunk 9]
```diff
@@ -11,6 +11,10 @@
 	}
 }
 
+.side-bar .site-title {
```
Comment (MattiSG) [inline _sass/custom/custom.scss]: I don't see this class exist anywhere.

File: assets/css/landing.css [hunk 10]
```diff
@@ -1,51 +1,103 @@
...
+}
 
-  -webkit-background-size: cover;
-  -moz-background-size: cover;
-  -o-background-size: cover;
-  background-size: cover;
+.landing .hero-head .logo {
+	position: absolute;
+    height: 60px;
+    width: auto;
```
Comment (MattiSG) [inline assets/css/landing.css]: Mixed whitespace.

> Please set up your editor appropriately.

File: assets/css/landing.css [hunk 11]
```diff
@@ -1,51 +1,103 @@
...
-	color: #363641;
+.our-work .work-item .title {
+	color: #FAFAFA;
 }
 
-.hero.is-info .box .subtitle {
-	color: #4a4a4a;
+.our-work .work-item .subtitle {
+	color: #FFF;
+    font-variant: all-small-caps;
```
Comment (MattiSG) [inline assets/css/landing.css]: Mixed whitespace (repeated below).

File: _pages/index.md [hunk 12]
```diff
@@ -3,53 +3,94 @@ layout: landing
...
 							Learn
 						</a>
 					</div>
 				</div>
 			</div>
 		</div>
-	</div>
-</section>
+	</section>
+	<footer class="footer">
```
Comment (MattiSG) [inline _pages/index.md]: Why duplicate the information through a class when the tag name is semantic enough? 🤔 
```suggestion
	<footer>
```

File: assets/css/landing.css [hunk 13]
```diff
@@ -1,51 +1,103 @@
...
+}
 
-.card-header {
-	flex-direction: column;
+.our-work .work-item .button {
+	margin-top: 1.5em;
+	min-width: 10em;
+	color: #4934A2;
+    font-weight: 400;
+    text-transform: uppercase;
```
Comment (MattiSG) [inline assets/css/landing.css]: Why do we need the `text-transform` when the content is already in uppercase?

File: _pages/index.md (hunk 2 above)
Comment (Ndpnt) [reply _pages/index.md]: Structure required by Bulma

File: _pages/index.md (hunk 4 above)
Comment (Ndpnt) [reply _pages/index.md]: I don't think it's necessary for now

File: _sass/custom/custom.scss (hunk 9 above)
Comment (Ndpnt) [reply _sass/custom/custom.scss]: It's generated by the theme

File: _pages/index.md (hunk 12 above)
Comment (Ndpnt) [reply _pages/index.md]: Required by Bulma CSS framework

File: assets/css/landing.css (hunk 13 above)
Comment (Ndpnt) [reply assets/css/landing.css]: Because the content is not already in uppercase

**Pull request merged**